libkbfs/disk_block_cache_remote.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"context"
+	"net"
+
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-framed-msgpack-rpc/rpc"
+	"github.com/keybase/kbfs/kbfsblock"
+	"github.com/keybase/kbfs/kbfscrypto"
+	kbgitkbfs "github.com/keybase/kbfs/protocol/kbgitkbfs"
```

Comment:
The package name actually is `kbgitkbfs1`, which is why I have to rename it. Not sure why the directory name is different.

Reply:
Yeah, was specifically asking about the directory.  I think it's supposed to have the 1 for versioning purposes, in case we ever completely revamp it in an incompatible way in the future. All the kbfs-server protocols have the suffix too...

Reply:
Not sure why it isn't generated that way. I'll check the code.